packages/mui-base/src/Badge/index.ts

```diff
@@ -1,7 +1,4 @@
 'use client';
-export { default } from './Badge';
-
+export { Badge } from './Badge';
 export * from './Badge.types';
-
-export { default as badgeClasses } from './badgeClasses';
 export * from './badgeClasses';
```

Comment:
This may be interesting use-case for the codemod to support. I assume it's not supported as I didn't see a test case for it.